x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/sender/RequestTask.java

```diff
@@ -131,17 +131,22 @@ public boolean isCancelled() {
         };
     }
 
-    private record Command(HttpClient httpClient, HttpUriRequest request, HttpClientContext context, ActionListener<HttpResult> listener)
-        implements
-            Runnable {
+    private record Command(
+        HttpClient httpClient,
+        HttpUriRequest requestToSend,
+        HttpClientContext context,
+        ActionListener<HttpResult> resultListener
+    ) implements Runnable {
 
         @Override
         public void run() {
             try {
-                httpClient.send(request, context, listener);
+                httpClient.send(requestToSend, context, resultListener);
             } catch (Exception e) {
-                logger.warn(format("Failed to send request [%s] via the http client", request.getRequestLine()), e);
-                listener.onFailure(new ElasticsearchException(format("Failed to send request [%s]", request.getRequestLine()), e));
+                logger.warn(format("Failed to send request [%s] via the http client", requestToSend.getRequestLine()), e);
```

Comment:
`RequestLine` contains the method and URI. I think this is ok to log as the URI is part of the public model configuration.